src/preset.ts

```diff
@@ -123,14 +122,9 @@ const observeAPIInfo = (interval: number, callback: (apiInfo: APIInfoPayload) =>
       callback({ aborted: true, connected: false });
       return;
     }
-    const ok = await fetch(CHROMATIC_API_URL, {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({ query: `{ viewer { id } }` }),
-    }).then(
-      (res) => res.ok,
-      () => false
-    );
+    const ok = await fetch("https://index.chromatic.com/healthz")
```

Comment:
I'm likely missing something, can you explain why this is necessary? Is it not a rather safe assumption that there will be a stable connection on a dev machine, and if not everything will be broken? What do we gain from constantly checking this status rather than handling failures in the exceptional case that there is no connectivity?

Reply:
Yeah that's a fair point. We could just do without the service status check and rely fully on the browser's online/offline event. In the unlikely event that our API goes down, we could just handle that in the existing GraphQL requests (to show the service status message on a 503 response).

Reply:
I ripped it out already.